src/pet.js

```diff
@@ -65,5 +66,14 @@ Pet.prototype.checkUp = function() {
         return 'I feel great!'
     }
```

Comment:
Props on the fact that you used the ternary operator in your conditionals! :100: